controllers/nstemplatetier/label_selector_test.go

```diff
@@ -33,6 +33,13 @@ func TestComputeHash(t *testing.T) {
 					},
 				},
 			},
+			Status: toolchainv1alpha1.NSTemplateTierStatus{
+				Revisions: map[string]string{
+					"base1ns-code-123456old":  "base1ns-code-123456old",
+					"base1ns-dev-123456old":   "base1ns-dev-123456old",
+					"base1ns-stage-123456old": "base1ns-stage-123456old",
+				},
+			},
```

Comment:
it looks like that this test whole test file doesn't actually verify the `OutdatedTierSelector` function, it looks more like a duplication of the tests that are in toolchain-common. I know that you didn't write the test, but could you please rewrite it? I can imagine that it could be a simple test calling the `OutdatedTierSelector` and using the selector to list Spaces (with multiple variations of the label), it doesn't have to be anything advanced.

Reply:
There are few test cases which check the above logic in [nstemplatier_space_mapper.go ](https://github.com/codeready-toolchain/host-operator/blob/508b798e82cbdd5b706fde35afbcbb671ffa130a/controllers/space/nstemplatetier_spaces_mapper_test.go#L25).
but i get your point, i have written the test cases which suits the label selector func

PTAL